virtcontainers/qemu_amd64.go

```diff
@@ -215,6 +216,8 @@ func (q *qemuAmd64) appendBridges(devices []govmmQemu.Device, bridges []Bridge)
 			t = govmmQemu.PCIEBridge
 		}
 
+		b.Addr = bridgePCIStartAddr + idx
```

Comment:
What happens on ARM?

Reply:
The code in qemu_arch_base will be used.